package.json

```diff
@@ -25,8 +25,8 @@
     "scripts": {
         "build": "tsc --build",
         "lint": "eslint . --ignore-path .gitignore",
-        "pretest": "npm run build",
-        "test": "npm run lint && karma start test/karma.config.cjs",
+        "pretest": "npm run lint",
+        "test": "wtr",
```

Comment:
Given the changes here, we'll need to update the ci workflow file to run `build` as a separate step, otherwise we won't have `tsc` coverage in CI any more.

I'd also be fine with dropping the `pretest` script entirely, though we will also need to add `npm run lint` to the ci workflow in that case as well.